jwala-common/src/main/java/com/cerner/jwala/common/rule/group/GroupNameRule.java

```diff
@@ -20,4 +20,9 @@ protected MessageResponseStatus getMessageResponseStatus() {
     protected String getMessage() {
         return "Invalid Group Name: \"" + name + "\"";
     }
+
+    @Override
+    public boolean isValid() {
+        return name != null && !"".equals(name.trim()) && name.matches("[A-Za-z0-9._\\s-]+");
```

Comment:
Please add a comment in the code to explain the characters allowed by the regular expression